### DISCUSSION TITLE
Web Radio import

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#178] [Fixed Issues] [answered]
@rern 

The Web Radio import process (update 202102051) seems to be truncating urls which contain an equals '=' sign. The attached PLS file has worked correctly in the past but now results in a Web Radio station with the url of:

http://as-hls-uk-live.bbcfmt.s.llnwi.net/pool_904/live/uk/bbc_radio_two/bbc_radio_two.isml/bbc_radio_two-audio

rather than the correct version:

http://as-hls-uk-live.bbcfmt.s.llnwi.net/pool_904/live/uk/bbc_radio_two/bbc_radio_two.isml/bbc_radio_two-audio=320000.norewind.m3u8

Chris
[BBC Radio 2.zip](https://github.com/rern/rAudio-1/files/5938669/BBC.Radio.2.zip)

**Comment by rern** [comment]
Fixed.

**Comment by ceejayemm** [comment, marked as answer]
Update 202102121 does indeed fix this problem.

Thanks

Chris
